src/set_theory/cardinal/ordinal.lean

```diff
@@ -696,6 +696,22 @@ eq.symm $ le_antisymm (mk_le_of_injective (λ x y, finset.singleton_inj.1)) $
 calc #(finset α) ≤ #(list α) : mk_le_of_surjective list.to_finset_surjective
 ... = #α : mk_list_eq_mk α
 
+@[simp] lemma mk_finsupp_of_infinite (α : Type u) (β : Type v) [infinite α] [has_zero β]
+  [nontrivial β] : #(α →₀ β) = max (lift.{v} (#α)) (lift.{u} (#β)) :=
```

Comment:
Please add a version for the case when both types are in the same universe so that one can easily rewrite from right to left in this case. Otherwise LGTM.
bors d+